list/to_c_or_not_to_c.md

```diff
@@ -182,3 +232,5 @@ Written by [ISSOtm](https://github.com/ISSOtm/) with help from [tobiasvl](https:
 I'm an active member of the GBDev community, which I joined about half a year ago. While that may seem small, my experience as a developer (check out my project, [Aevilia](https://github.com/ISSOtm/Aevilia-GB/)) and as a moderator of the Discord server, has allowed me to interact with RGBDS and GBDK users alike.
```

Comment:
If we manage to unbias this, we could remove this note and just mention the main contributors, effectively presenting the document as "gbdev community" endorsed (which I'd really like), instead of a private post from ISSO.

Reply:
Sounds good if ISSO's is in favor of it